src/test/java/core/basesyntax/ReadCsvFileServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.ReadFileException;
+import core.basesyntax.service.ReadCsvFileService;
+import core.basesyntax.service.implementations.ReadCsvFileServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class ReadCsvFileServiceTest {
+    private static String SOURCE_FILE = "src/test/resources/NormalTestInput.csv";
+    private static String NOT_EXIST_SOURCE_FILE = "src/test/resources/nonexistent-file.csv";
+    private static ReadCsvFileService readCsvFileService;
+
+    @BeforeAll
+    static void setReadService() {
+        readCsvFileService = new ReadCsvFileServiceImpl();
+    }
+
+    @Test
+    void readFile_validData_okay() {
+        List<String> data = assertDoesNotThrow(() -> readCsvFileService.readFile(SOURCE_FILE));
+        try {
```

Comment:
You should assert that data you red is equal to expected data

assertEquals(data_you_read, some list that represents expected data not from file as you did now, just list)